AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -1259,6 +1259,7 @@ zerohedge.com##div[class^="TheMarketEarHomePageSidebar_container__"]
 cyberleninka.ru#?#.infoblock:has(> div[class="full"] > div.bibloid_serp)
 drwindows.de#?#section[class^="artikel-"] > ul > li:not([class]):has(> a[href] > h3.post:contains(Anzeige))
 stopgame.ru##.corner_banner
+fantia.jp###post-footer
```

Comment:
I don't think we should block the whole element entirely, just `share` part. @Yuki2718 please also take a look.

Reply:
I agree, either extended syntax or `###post-footer > div.well-overlay-white:last-child`